devtools/spellcheck/spellcheck.sh

```diff
@@ -39,12 +39,12 @@ src="$( cd -P "$( dirname "${BASH_SOURCE[0]}" )/../.." && pwd )"
 
 output=$(/usr/bin/find "$src/XenAdmin" "$src/XenModel" "$src/XenOvfApi" "$src/XenOvfTransport" \( -name \*.cs -o -name \*.resx \) -exec sh "$dir/spellcheck-file.sh" \{\} \;)
 
-output_brand=$(/usr/bin/find "$src/XenAdmin" "$src/XenModel" "$src/XenOvfApi" "$src/XenOvfTransport" \( -name \*.resx \) -exec sh "$dir/spellcheck-brand-file.sh" \{\} \;)
+output_brand=$(/usr/bin/find "$src/XenAdmin" "$src/XenModel" "$src/XenModel/XenAPI" "$src/XenOvfApi" "$src/XenOvfTransport" \( -name \*.resx \) -exec sh "$dir/spellcheck-brand-file.sh" \{\} \;)
```

Comment:
Why was "$src/XenModel/XenAPI" added here? Is it not covered by "$src/XenModel" (as I assume we run the check in the subfolfers as well)?

Reply:
A mistake :). I thought subfolders were not covered, so I added it to check FriendlyErrorNames.resx. I confirmed, it works on subfolders.